ksqldb-tools/src/test/java/io/confluent/ksql/tools/migrations/util/CommandParserTest.java

```diff
@@ -296,7 +296,7 @@ public void shouldParseCreateConnectorStatement() {
   @Test
   public void shouldParseDropConnectorStatement() {
     // Given:
-    final String dropConnector = "DROP CONNECTOR `jdbc-connector`;";
+    final String dropConnector = "DROP CONNECTOR `jdbc-connector` ;";
```

Comment:
Let's add a comment so future readers know the motivation for including a space before the semicolon here, since the intention isn't obvious otherwise.